src/main/frontend/dicts.cljs

```diff
@@ -3051,7 +3051,7 @@
            :settings-page/show-brackets "Mostrar parênteses retos"
            :settings-page/spell-checker "Verificador ortográfico"
            :settings-page/disable-sentry "Enviar dados de utilização e diagnósticos para Logseq"
-           :settings-page/preferred-outdenting "Outdenting lógico"
+           :settings-page/preferred-outdenting "Desindentação lógica"
```

Comment:
Indentação

Reply:
"**Indenting**" isn't the same as "**Outdenting**". Sadly, there's no word for "**outdenting**" in Portuguese (maybe "recuo"?). I think in this case the closest possible thing to it is "**desindentar**" (the opposite of "**indenting**"), which isn't widely used but there are some use cases of the word as it technically is a newly correctly formed word using derivation (derivação do prefixo des + indentar). Hope you understand this choice of words.